packages/query/tests/collection-replication-state.test.ts

```
@@ -1252,6 +1252,97 @@ describe('CollectionReplicationState', () => {
+				date_modified_gmt: '2026-08-03T12:00:05',
+			});
+			const originalRevision = optimisticDoc.revision;
+			let resolvePost!: (value: unknown) => void;
+			httpClientMock.post.mockImplementationOnce(
+				() => new Promise((resolve) => (resolvePost = resolve))
+			);
+
+			const createPromise = replicationState.remoteCreate(
+				optimisticDoc.toJSON(),
+				originalRevision
+			);
```

ESLint

[error] Replace `⏎↹↹↹↹optimisticDoc.toJSON(),⏎↹↹↹↹originalRevision⏎↹↹↹` with `optimisticDoc.toJSON(),·originalRevision`

(prettier/prettier)